app/renderer/js/utils/request-util.ts

```diff
@@ -32,7 +32,7 @@ class RequestUtil {
 		if (certificate) {
 			// To handle case where certificate has been moved from the location in certificates.json
 			try {
-				certificateLocation = fs.readFileSync(certificate);
+				certificateLocation = fs.readFileSync(certificate).toString();
```

Comment:
Is this needed, `toString()`?

Reply:
Yes, you need to do that otherwise this would be a buffer. Another way which is think makes more sense is to pass `utf8` as the encoding parameter.